### ISSUE TITLE
Interface crashes when using conditions inside of the image_map field type 

### ISSUE THREAD

**Issue by thomasduenser** [sulu/sulu#6261]
| Q | A
| --- | ---
| Sulu Version | 2.3.5 (also tested in 2.3.4)
| PHP Version | 7.4.16
| DB Version | MySQL 5.7
| Browser Version | Chrome 94.0.4606.61

# Actual Behavior

A white screen appears when the user tries to add a hotspot for the content type "image_map".

![Bildschirmfoto 2021-10-01 um 08 26 28](https://user-images.githubusercontent.com/3875917/135576543-0cd3fa7c-a800-4a56-bacb-60d0469deb2c.png)

# Expected Behavior

JS shouldn't break.

# Steps to Reproduce

Use following content type in a template and try to add a hotspot to one of the images.

```xml
<?xml version="1.0" ?>
<type xmlns="http://schemas.sulu.io/template/template"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
      xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd"
      name="interactive-image">
    <meta>
        <title lang="en">Interactive image</title>
        <title lang="de">Interaktives Bild</title>
    </meta>

    <properties>
        <block name="interactives" maxOccurs="5" default-type="interactive">
            <types>
                <type name="interactive">
                    <meta>
                        <title lang="en">Interactive image</title>
                        <title lang="de">Interaktives Bild</title>
                    </meta>

                    <properties>
                        <property name="imageMap" type="image_map" default-type="basic">
                            <meta>
                                <title lang="en">Image-Map</title>
                                <title lang="de">Image-Map</title>
                            </meta>

                            <types>
                                <type name="basic">
                                    <meta>
                                        <title lang="en">Basic</title>
                                        <title lang="de">Grundlegend</title>
                                    </meta>

                                    <properties>
                                        <property name="internalLinks" type="page_selection">
                                            <meta>
                                                <title lang="en">Internal link</title>
                                                <title lang="de">Interner Link</title>
                                            </meta>

                                            <params>
                                                  <param name="properties" type="collection">
                                                    <param name="excerptMore" value="excerpt.more"/>
                                                </param>
                                            </params>
                                        </property>
                                    </properties>
                                </type>
                            </types>
                        </property>
                    </properties>
                </type>
            </types>
        </block>
    </properties>
</type>
```

**Comment by thomasduenser** [comment]
The issue is resolved for my case. We had visible conditions on two properties which caused this issue - maybe they were wrong as we should've used the new `__parent` variable? 🤔 

```
visibleCondition="'applications-overview' != template"
visibleCondition="'applications-overview' == template"
```

**Comment by niklasnatter** [comment]
You need the `__parent` variable only if you want to access data from the same block. If you want to access a root property, you dont need to use `__parent` 🙂 

Can you reproduce the problem by adding your `visibleCondition` to another template?

**Comment by niklasnatter** [comment]
After some investigation, this looks like a bug that occurs when using a `visibleCondition`/`disabledCondition` inside of a `image_map`. The problem is most likely related to #5705 and therefore should be present in all `2.3.*` versions.

I can reproduce the problem by adding this property to my template:
```xml
<property name="image_map" type="image_map" default-type="basic">
    <meta>
        <title lang="en">Image-Map</title>
        <title lang="de">Image-Map</title>
    </meta>

    <types>
        <type name="basic">
            <meta>
                <title lang="en">Basic</title>
                <title lang="de">Grundlegend</title>
            </meta>

            <properties>
                <property name="text_editor" type="text_editor" visibleCondition="false">
                    <meta>
                        <title lang="en">Text Editor</title>
                        <title lang="de">Texteditor</title>
                    </meta>
                </property>
            </properties>
        </type>
    </types>
</property>
```

**Comment by niklasnatter** [comment]
Fixed in #6264
